tests/test_ws.py

```
@@ -0,0 +1,40 @@
+
+        # We expect at least one chunk and a 'done' message
+        # Note: In test environment, OllamaProcessor might fail or return mock
+        # depending on how it's implemented.
+        try:
+            response = websocket.receive_json()
+            assert response["type"] in ["chunk", "error"]
+
+            if response["type"] == "chunk":
+                done = websocket.receive_json()
+                assert done["type"] == "done"
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)